libr/util/list.c

```diff
@@ -30,6 +30,11 @@ R_API RListIter *r_list_iter_get_next(RListIter *list) {
 	return list->n;
 }
 
+R_API RListIter *r_list_iter_get_prev(RListIter *list) {
```

Comment:
define it inline in r_list.h

Reply:
ping ^*

Reply:
why? `r_list_iter_get_next` also exists here